Bindings/Python/README.md

```diff
@@ -12,7 +12,7 @@ Release Build
 cd Bindings\Python
 mkdir build
 cd build
-cmake -A x64 ..
+cmake ..
```

Comment:
Right now my current build env for this project is VS2017.  From what I've read, if you are building with VS2017 it defaults to Win32 so you have to somehow specify you are targetting an x64 system.  It might be easier to document that for VS2017, as they will want to use "cmake -A x64 ..", unless there is someway to always force that in the CMakeLists.txt file.

Reply:
I see, I removed it because I was getting the following error message in Linux:
```
CMake Error at CMakeLists.txt:2 (project):
  Generator

    Unix Makefiles

  does not support platform specification, but platform

    x64

  was specified.


CMake Error: CMAKE_C_COMPILER not set, after EnableLanguage
CMake Error: CMAKE_CXX_COMPILER not set, after EnableLanguage
-- Configuring incomplete, errors occurred!
```

One other option I think would be to use CMAKE_GENERATOR_PLATFORM. As far as I understand it:
> For Visual Studio Generators with VS 2005 and above this specifies the target architecture.
https://cmake.org/cmake/help/latest/variable/CMAKE_GENERATOR_PLATFORM.html

You would need to test that though, I don't know for sure if that works. 
There are also MSVC compiler options, such as /favor:AMD64. 

On my Windows machine it is actually selecting x64 by default, so I cannot really test it
```
-- Building for: Visual Studio 16 2019
-- Selecting Windows SDK version 10.0.18362.0 to target Windows 10.0.19042.
-- The C compiler identification is MSVC 19.27.29112.0
-- The CXX compiler identification is MSVC 19.27.29112.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/BuildTools/VC/Tools/MSVC/14.27.29110/bin/Hostx64/x64/cl.exe - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/BuildTools/VC/Tools/MSVC/14.27.29110/bin/Hostx64/x64/cl.exe - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found PythonInterp: C:/Users/Paul/miniconda3/python.exe (found version "3.8.3")
-- pybind11 v2.6.2 dev1
-- Found PythonLibs: C:/Users/Paul/miniconda3/libs/python38.lib
-- Performing Test HAS_MSVC_GL_LTCG
-- Performing Test HAS_MSVC_GL_LTCG - Success
-- Configuring done
-- Generating done
-- Build files have been written to: C:/Users/Paul/Documents/Doxa/Bindings/Python/build
```

Reply:
That will do it!  I'll merge this and then check in a tweak for windows that should get it working.